.travis.yml

```diff
@@ -63,25 +58,15 @@ before_install:
 
     git config --global advice.detachedHead false
 
-    # Export tests environment variables
```

Comment:
Could we move this to `.ci` directory which is at the project root. I plan to migrate all provision/build files from `tests/_ci` to `.ci`